<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Praaline::Plugins::StatisticsPluginPCA::PCAPlotWidget</class>
 <widget class="QWidget" name="Praaline::Plugins::StatisticsPluginPCA::PCAPlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCA Plot</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutWidget">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QDockWidget" name="dockWidgetPlot">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="windowTitle">
       <string>Plot</string>
      </property>
      <widget class="QWidget" name="dockWidgetPlotContents">
       <layout class="QGridLayout" name="gridLayoutPlot">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QDockWidget" name="dockWidgetParameters">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="windowTitle">
       <string>Parameters</string>
      </property>
      <widget class="QWidget" name="dockWidgetParametersContents">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayoutAttributeComboBoxes">
          <item row="1" column="0">
           <widget class="QLabel" name="labelClassificationAttribute">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Classification attribute:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelFilterAttribute">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Filter attribute:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="comboBoxClassificationAttribute">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QComboBox" name="comboBoxFilterAttribute">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listFilterAttributeValues">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBoxActionOnClick">
          <property name="title">
           <string>Action upon double-clicking on a Corpus Item</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QRadioButton" name="optionActionPlay">
             <property name="text">
              <string>Play Corpus Recording</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="optionActionVisualiser">
             <property name="text">
              <string>Open in Visualiser</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="optionActionNothing">
             <property name="text">
              <string>Do nothing</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelSelectedItem">
          <property name="text">
           <string>Selected item properties:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QTreeWidget" name="treeWidgetItemProperties">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxClassificationAttribute</tabstop>
  <tabstop>comboBoxFilterAttribute</tabstop>
  <tabstop>listFilterAttributeValues</tabstop>
  <tabstop>optionActionPlay</tabstop>
  <tabstop>optionActionVisualiser</tabstop>
  <tabstop>optionActionNothing</tabstop>
  <tabstop>treeWidgetItemProperties</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
